castai/resource_rebalancing_schedule.go

```diff
@@ -95,7 +96,7 @@ func resourceRebalancingSchedule() *schema.Resource {
 							ValidateDiagFunc: validation.ToDiagFunc(validation.IntAtLeast(1)),
 							Description:      "Minimum number of nodes that should be kept in the cluster after rebalancing.",
 						},
-						"keep_drain_timeout_nodes": {
```

Comment:
This is a breaking change. We shouldn't just remove the filed — first deprecation and than remove.